scripts/cybershake/queue_monitor.py

```diff
@@ -151,6 +153,49 @@ def update_tasks(
                     "Disappeared from squeue. Creating a new task.",
                 )
             )
+            # When job failed, we want to log metadata as well
```

Comment:
Should probably be it's own function?
Could eventually be used for recording both failed and completed jobs

Reply:
yes, as we discussed in the meeting, its probably a good idea to remove the metadata logging function from the *.sl and only monitor will log them.
But no matter what way to are going to use, the functionality to query and log metadata needs to be modularized so it can be called wither within code or outside ( as @jasonmotha mentioned in his comment)